File: Python/01 - 10/06 - eulerSix.py
def main():
    sum_squared, sum = 0, 0
    for i in range(0, 101, 1):
        sum_squared += i
    for j in range(0, 101, 1):
        numSquared = j * j
        sum += numSquared
    sum_squared *= sum_squared
    diffrence = sum_squared - sum
    print("The difference between the sum of the squares of the first one hundred natural numbers and the square of the sum is:", diffrence)

if __name__ == "__main__":
    main()

File: C++/01 - 10/06_eulerSix.cpp
// Find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum.
#include <iostream>
int main() {
    int sumOfSquares = 0, sum = 0;
    for(int i = 1; i <= 100; i++) {
        sumOfSquares += i * i;
        sum += i;
    }
    sum *= sum;
    std::cout << sum - sumOfSquares << " is difference between the sum of the squares of the first one hundred natural numbers and the square of the sum.\n";
    return 0;
}